tests/orquestra/quantum/circuits/_serde_test.py

```diff
@@ -128,6 +128,12 @@
             _builtin_gates.RY(PARAMETER_VECTOR[0] * PARAMETER_VECTOR[1])(1),
         ]
     ),
+    _circuit.Circuit(
+        [
+            _builtin_gates.Z.power(0.25)(0),
+            _builtin_gates.SWAP.power(0.5)(0, 1),
+        ]
```

Comment:
Good addition here!

Reply:
Thanks! Beyond testing the serialization, I chose those specific gates to confirm that `Power` gates could be constructured corresponding to the examples you mentioned in the issue.